docs/ref/contrib/postgres/functions.txt

```diff
@@ -0,0 +1,31 @@
+======================================
+PostgreSQL-specific Database Functions
+======================================
+
+.. module:: django.contrib.postgres.functions
+    :synopsis: PostgreSQL-specific Database Functions
+
+.. versionadded:: 1.9
+
+The functions documented on this page allow users to access PostgreSQL-specific
+database functions to be used in annotations in Django.
+
+Similarity
+----------
+
+.. class:: Similarity(expression, expression, **extra)
+
+Accepts a field name or expression, and a string or expression. Returns
+the similarity measurement between the two arguments.
+
+Usage example::
+
+    >>> from django.contrib.postgres.functions import Similarity
+    >>> Author.objects.create(name='Katy Stevens')
+    >>> author = Author.objects.annotate(
+    ...    similarity=Similarity('name', 'Katie Stephens')).get()
+    >>> print(author.similarity)
+    0.4
+
+You could sort results by the returned similarity descending, in order to
```

Comment:
Why not just do the example like this? A more realistic example would be to only accept values over a certain value (0.3 being the natural choice as this is what `%%` does)

For example, try having another author called `Stephen Keats`